File: src/main/java/com/talenco/lotterysystem/service/EmployeeService.java
package com.talenco.lotterysystem.service;

import com.talenco.lotterysystem.POJO.Employee;

import java.util.List;

public interface EmployeeService {

    List<Employee> getAll();

    Employee getOne(Long id);

    void insert(Employee employee) throws Exception;

    void update(Employee employee);

    void delete(Long[] ids);
}

File: src/main/webapp/js/service/employee_service.js
app.service('employeeService',function($http){


    this.getAll=function(){
        return $http.get('../EmployeeController/getAll');
    };

    this.getOne=function(empno){
        return $http.get('../EmployeeController/getOne?empno=' + empno);
    };

    this.insert=function(entity){
        return $http.post('../EmployeeController/insert',entity);
    };

    this.dele=function(selectIds){
        if(confirm("你确认要删除么")) {
            return $http.get('../EmployeeController/delete?ids=' + selectIds);
        }
    };

    this.update=function(entity){
        return $http.post('../EmployeeController/update',entity);
    }
});